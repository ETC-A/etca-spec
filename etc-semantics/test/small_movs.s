;
;0
    mov %r0, 1
    mov %r1, 2
    mov %r2, 3
    mov %r3, 4
    mov %r4, 5
    mov %r5, 6
    mov %r6, 7
    mov %r7, 8
end:
    jmp end