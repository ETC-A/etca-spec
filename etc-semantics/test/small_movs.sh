set -u
./scripts/test.sh "$1" "$2" "$3" "test/small_movs.bin" 0
