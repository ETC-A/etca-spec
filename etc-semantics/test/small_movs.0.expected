<etc>
  <k>
    #halt "Halted successfully!" ~> .
  </k>
  <pc> 32784 </pc>
  <registers>
    0 |-> 1
    1 |-> 2
    2 |-> 3
    3 |-> 4
    4 |-> 5
    5 |-> 6
    6 |-> 7
    7 |-> 8
  </registers>
  <memory>
    128 |-> b"Y\x01Y\"YCYdY\x85Y\xa6Y\xc7Y\xe8\x8e\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
  </memory>
  <flags>
    <carry>    false </carry>
    <zero>     false </zero>
    <negative> false </negative>
    <overflow> false </overflow>
  </flags>
  <machine-details>
    <cpuid1> 0 </cpuid1>
    <cpuid2> 0 </cpuid2>
    <feat>   0 </feat>
    <reg-width> word </reg-width>
    <reg-count> 8 </reg-count>
    <evil-mode> false </evil-mode>
  </machine-details>
  <in> </in>
  <out> </out>
</etc>
